src/components/layout.js

```diff
@@ -14,15 +14,15 @@ import Navigation from "../sections/General/Navigation";
 import Footer from "../sections/General/Footer";
 import { GlobalStyle } from "../sections/app.style";
 
-const Layout = ({ children }) => {
+const Layout = ({ children,location }) => {
 
   return (
     <>
       <GlobalStyle />
       <Navigation/>
       {children}
       <ScrollToTopBtn />
-      <Footer />
+      <Footer location={location} />
```

Comment:
Exactly, what I was trying to say in my comment on the other PR :)

Reply:
Yes, I also completely missed the method.

Thanks to your comments, I re-examined my assumptions, took a more careful look at what was happening, and it showed the path(name). 

Cheers,
Randy